policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_exemptions_indiv.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class mt_exemptions_indiv(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana exemptions when married couples file separately"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period).astype(
```

Comment:
dont need astype(int)

Reply:
I believe we do, as the following test fails without:
```
- name: The exemptions are allocated to the head
  period: 2022
  input:
    people:
      person1:
        is_tax_unit_head: true
        is_blind: true
      person2:
        is_tax_unit_spouse: true
        mt_aged_exemption_eligible_person: true
      person3:
        mt_dependent_exemptions: true
    households:
      household:
        members: [person1, person2, person3]
        state_code: MT
  output:
    mt_exemptions_indiv: [8_130, 5_420, 0]
```